build_projects/shared-build-targets-utils/VersionRepoUpdater.cs

```diff
@@ -60,6 +60,19 @@ public class VersionRepoUpdater
             await UpdateGitHubFile(packageInfoFilePath, packageInfoFileContent, message);
         }
 
+        public async Task PublishVersionBadge(string badgePath, string channel, string pointer)
+        {
+            byte[] data = System.IO.File.ReadAllBytes(badgePath);
+            string githubFileName = $"{s_baseGitHubPath}/{channel}/{pointer}/{Path.GetFileName(badgePath)}";
+            await PushChangesToGitHubFile(githubFileName, data, "Updating CLI version badge");
```

Comment:
These messages should have the version number in them, if possible.  This makes the commit history a lot easier to consume.